core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -233,12 +252,56 @@ public static boolean defaultUseHttp1Pipelining() {
      * {@code defaultBackoffSpec} in the constructor call.
      *
      * <p>The default value of this flag is {@value DEFAULT_DEFAULT_BACKOFF_SPEC}. Specify the
-     * {@code -Dcom.linecorp.armeria.defaultBackoffSpec=<spec>} to override the default value.
+     * {@code -Dcom.linecorp.armeria.defaultBackoffSpec=<spec>} JVM option to override the default value.
      */
     public static String defaultBackoffSpec() {
         return DEFAULT_BACKOFF_SPEC;
     }
 
+    /**
+     * Returns the value of the {@code routeCache} parameter. It would be used to create a route {@link Cache}
```

Comment:
Could you add the offline link to Caffeine Javadoc?

https://github.com/line/armeria/blob/master/build.gradle#L914

You'll have to add the package list file to the `site/src/apidocs/package-lists` directory.